Chaincase.Common/Global.cs

```diff
@@ -428,14 +416,10 @@ public async Task OnSleeping()
                     IsGoingToSleep = true;
 
                     // don't ever cancel Init. use an ephemeral token
-                    await WaitForInitializationCompletedAsync(new CancellationToken());
+                    await InitializationCompleted.Task;
 
-
-                    if (_torManager?.State != TorState.Stopped) // OnionBrowser && Dispose@Global
-                    {
-                        await _torManager.StopAsync();
-                        Logger.LogInfo($"{nameof(Global)}.OnSleeping():{nameof(_torManager)} is stopped.");
-                    }
+                    await _torManager.StopAsync(SleepCts.Token);
```

Comment:
Same here, vestige of https://github.com/OnionBrowser/OnionBrowser/blob/0f835926946c2b6c084098417a58fa59ecc3c98a/OnionBrowser/AppDelegate.swift#L280-L287

I'm not sure we want to remove the conditional